gpep517/__main__.py

```diff
@@ -130,6 +130,22 @@ def install_wheel(args):
     return 0
 
 
+def verify_pyc(args):
+    from gpep517.qa import qa_verify_pyc
+
+    install_dict = install_scheme_dict(args.prefix or "/usr", "")
+    sitedirs = frozenset(install_dict[x] for x in ("purelib", "platlib"))
+    result = qa_verify_pyc(args.destdir, sitedirs)
+
+    def fpath(p):
+        return "/" + os.path.relpath(p, args.destdir)
+
+    for kind, entries in result.items():
+        for e in entries:
+            print(f"{kind}:{':'.join(fpath(x) for x in e)}")
+    return 1 if any(v for v in result.values()) else 0
```

Comment:
I don't know if this would be nicer, but you can do:
```suggestion
    return int(any(v for v in result.values()))
```

Reply:
Yeah, I actually like the explicit intent more.